crates/accelerate/Cargo.toml

```
@@ -45,3 +45,10 @@ features = ["rayon"]
 [dependencies.indexmap]
 version = "2.1.0"
 features = ["rayon"]
+
+[dependencies.faer]
+version = "0.16.0"
+features = ["ndarray", "nalgebra"]
```

I tested this locally we can remove `nalgebra` here, and since nothing in qiskit is using nalgebra this saves us a few build dependencies.

```suggestion
features = ["ndarray"]
```

---

Would be nice if there were a lint for this.